packages/web-components/src/dropdown/README.md

```diff
@@ -0,0 +1,176 @@
+| `proxy`           | public  |                                                                                                                  |           |                                                      |
+| `selectedIndex`   | public  | `number`                                                                                                         | `-1`      | The index of the selected option.                    |
+| `selectedOptions` | public  | `FluentListboxOption[]`                                                                                          | `[]`      | A collection of the selected options.                |
+| `size`            | public  | `{ small: 'small', medium: 'medium', large: 'large' }`                                                           | `medium`  | The maximum number of options to display.            |
+| `value`           | public  |                                                                                                                  |           | The value property.                                  |
+
+<br />
+
+### **Methods**
+
+| Name                 | Privacy | Description                                                                | Parameters                                  | Return | Inherited From                                                                                                                   |
+| -------------------- | ------- | -------------------------------------------------------------------------- | ------------------------------------------- | ------ | -------------------------------------------------------------------------------------------------------------------------------- |
+| `multipleChanged`    | public  | Sets the multiple property on the proxy element.                           | `prev: boolean or undefined, next: boolean` |        |                                                                                                                                  |
+| `selectFirstOption`  | public  | Moves focus to the first selectable option.                                |                                             | `void` | [FASTListbox](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-foundation/src/listbox/listbox.spec.md) |
+| `setPositioning`     | public  | Calculate and apply listbox positioning based on available viewport space. |                                             | `void` |                                                                                                                                  |
+| `setSelectedOptions` | public  | Sets an option as selected and gives it focus.                             |                                             |        | [FASTListbox](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-foundation/src/listbox/listbox.spec.md) |
```

Comment:
Some of these are "methods" and some are attribute changed callbacks. The intent of each is different. A method would be, "Use `MyComponent.open()` to open the component....". The attribute changed callback is intended to do work when an attribute changes and for the most part we try to keep those as protected so they can primarily be used when extending the class.